contentful/export-processor/content-types/section.js

```diff
@@ -152,6 +152,119 @@ export class Section {
     return minimumPathsForRecommendations;
   }
 
+  /**
+   * Calcutes paths to select each answer to each question
+   */
+
+    getPathsForAllAnswers() {
+        const allAnswerPaths = []
+        const allAnswers = []
+        const answersUsed = []
+
+        // Log ids of all answers in section
+        this.questions.forEach((question) => {
+            question.answers.forEach((answer) => {
+                allAnswers.push(answer.id)
+            })
+        })
+
+        // Log ids of answers used in minimum paths for recommendations
+        const allRecommendationPaths = this.minimumPathsForRecommendations;
+        for (const maturity in allRecommendationPaths) {
+            allRecommendationPaths[maturity].forEach((question) => {
+                if (!answersUsed.includes(question.answer.id)) {
+                    answersUsed.push(question.answer.id)
+                }
+            })
+        }
+
+        // Create paths until all answers are used
+        while (answersUsed.length !== allAnswers.length) {
+            const neededNextQuestions = [];
+            const newPath = [];
+            const lastQuestion = this.questions[this.questions.length - 1];
+            let currentAnswer;
+
+            // Add last question and answer
+            const unusedLastAnswers = lastQuestion.answers.filter(answer => !answersUsed.includes(answer.id))
+
+            if (unusedLastAnswers.length) {
+                newPath.unshift({ question: lastQuestion, answer: unusedLastAnswers[0] })
+                answersUsed.push(unusedLastAnswers[0].id)
+                neededNextQuestions.unshift(lastQuestion.id);
+            } else {
+                newPath.unshift({ question: lastQuestion, answer: lastQuestion.answers[0] })
+            }
+
+            // Loop through remaining questions
+            for (let i = this.questions.length - 2; i >= 0; i--) {
+                const currentQuestion = this.questions[i]
+                if (neededNextQuestions.length) {
+                    const validAnswers = currentQuestion.answers.filter(answer => answer.nextQuestion && answer.nextQuestion.sys.id == neededNextQuestions[0])
```

Comment:
is it possible that valid answers will be empty in the scenario that the only way to reach question 6, is via an answer from question 4? (for example) because this will go to q5 and if no q5 answers lead to q6 it could get stuck

I think the potential way to get around it, is to decrement i until you find an answer that leads to the `neededNextQuestion`